index.js

```diff
@@ -0,0 +1,88 @@
+import { DateTime } from './modules/luxon.js';
+import Book from './modules/storage.js';
+import displayBooks from './modules/displaybook.js';
+import hideSections from './modules/hidesection.js';
+
+const submitButton = document.getElementById('add');
+const displayBookList = document.getElementById('table');
+
+const bookTitle = document.getElementById('book-title');
+const author = document.getElementById('author');
+const error = document.getElementById('error');
+const form = document.getElementById('form');
+
+const listSection = document.getElementById('list');
+const newBookSection = document.getElementById('book-form');
+const contactSection = document.getElementById('contact-info');
+
+const listLink = document.getElementById('list-link');
+const addNewLink = document.getElementById('add-new-link');
+const contactLink = document.getElementById('contact-link');
+
+const displayDate = () => {
+  const date = document.getElementById('current-date');
+  date.innerHTML = DateTime.now();
+};
```

Comment:
[OPTIONAL] I would encourage you to organize the date in an organized way so that it would be readable.

Reply:
@Olamarx Thank you so much for pointing this out. Solving this issue makes the date more readable now.